main/FatesInterfaceTypesMod.F90

```diff
@@ -331,6 +336,10 @@ module FatesInterfaceTypesMod
                                                  ! be equivalent (ie integer ascending)
                                                  ! Or, all will be 1.
       
+     integer,allocatable   :: sp_patch_index(:)  ! in SP mode, we need to map the p values for each patch
```

Comment:
I do not find any usage of this variable.....

Reply:
It was a remnant of a previous attempt to solve the SP mode indexing issues. Luckily it can now be deleted. I think/hope :)